website/content/partials/pki-double-migration-bug.mdx

```diff
@@ -0,0 +1,22 @@
+### PKI Storage Migration Revives Deleted Issuers
+
+In Vault 1.11, a new storage layout was created to support multiple issuer
+within a single mount, creating storage version 1. In Vault 1.11.6, 1.12.2,
+and 1.13.0, a bug was fixed wherein (due to incorrect ordering of writes),
+the CA chain could be incorrect, due to writes failing when too many mounts
+attempted migration at the same time; this introduced storage version 2.
+
+#### Impacted Versions
+
+If issuers were present pre-1.11, and were subsequently deleted prior to
+upgrading to a Vault version with storage version 2, the initial migration
+of issuers would be re-attempted and these issuers would be recreated. If
+managed keys were associated with these legacy versions (and were
+subsequently removed from the managed key repository), migration would fail
+with an error in the logs:
```

Comment:
```suggestion
Issuers saved under Storage v1 that were deleted before upgrading to a Vault
version with Storage v2 were incorrectly recreated during the upgrade when
Vault automatically migrated legacy issuers. Migration failed if Vault
found managed keys associated with the legacy issuers that had been removed
from the managed key repository. The error appeared in Vault logs as:
```

Edited for accessibility (active voice, shorter sentences)

Reply:
I think this one is actually incorrect... 

```suggestion
Issuers created before Vault 1.11 and migrated to Storage v1 and deleted
before upgrading to a Vault version with Storage v2 were incorrectly
recreated during the upgrade when Vault automatically re-migrated legacy
issuers. Migration failed if Vault found managed keys associated with the
legacy issuers that had been removed from the managed key repository. The
error appeared in Vault logs as:
```